finance/sc-im/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           makefile 1.0
+
+github.setup        andmarti1424 sc-im 0.8.2 v
+maintainers         @nkorth openmaintainer
+license             BSD
+categories          finance
+description         Spreadsheet Calculator Improvised
+long_description    An ncurses spreadsheet program for terminal
+
+platforms           darwin
+
+github.tarball_from archive
+checksums           rmd160  afabf063feda1800fd07e6ee13b4036c7d5a200f \
+                    sha256  7f00c98601e7f7709431fb4cbb83707c87016a3b015d48e5a7c2f018eff4b7f7 \
+                    size    1619617
+
+worksrcdir          ${distname}/src
+makefile.prefix_name prefix
```

Comment:
This isn't working, I'm getting `Warning: violation by /usr` (which wasn't happening before switching to PortGroup makefile)